### ISSUE TITLE
Support for indented syntax (.sass)

### ISSUE THREAD

**Issue by zezic** [bajrangCoder/zed-scss#3]
Would be good to support https://sass-lang.com/documentation/syntax/#the-indented-syntax

**Comment by bajrangCoder** [comment]
Of course, in the initial version, I omitted it under the assumption that the indent syntax was not widely used. However, I plan to include it in the upcoming version.

Thanks for your feedback ♥️

**Comment by bajrangCoder** [comment]
This will be a separate extension.

**Comment by miphe** [comment]
Would be great to add indented syntax support. As far as I know this is definitely widely used.

**Comment by narmatoff** [comment]
Bro, we are really looking forward to it! For me personally, this is the only stop factor for the full use of zed...
